core/components/popular-products/index.tsx

```
@@ -0,0 +1,101 @@
+import { Streamable } from '@/vibes/soul/lib/streamable';
+import { type Product } from '@/vibes/soul/primitives/product-card';
+import { ProductList } from '@/vibes/soul/sections/product-list';
+import { SectionLayout } from '@/vibes/soul/sections/section-layout';
+import { ArrowButton } from '~/components/arrow-button';
+import { Link } from '~/components/link';
+
+interface Link {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Link'. Consider to delete it or rename it.

'Link' is defined here:


(lint/suspicious/noRedeclare)